stdlib/public/Python/Python.swift

```diff
@@ -228,14 +228,20 @@ extension PythonError : Equatable {
 extension PythonError : CustomStringConvertible {
   public var description: String {
     switch self {
-    case .exception(let p): return "exception: \(p)"
-    case .invalidCall(let p): return "invalidCall: \(p)"
-    case .invalidMember(let m): return "invalidMember: \(m)"
-    case .invalidModule(let m): return "invalidModule: \(m)"
+    case .exception(let p): return "Python Exception: \(p)"
```

Comment:
Nit-pick: personally I'd prefer that only `Python` is capitalized in the description:
`"Python exception: \(p)"`, `"Python invalid call: \(p)"`, etc.